var/spack/repos/builtin/packages/npm/package.py

```diff
@@ -9,13 +9,18 @@
 from spack.package import *
 
 
-# NOTE: not actually an Autotools package
 class Npm(Package):
     """npm: A package manager for javascript."""
 
     homepage = "https://github.com/npm/cli"
     # base https://www.npmjs.com/
-    url = "https://registry.npmjs.org/npm/-/npm-6.13.4.tgz"
+
+    git = "https://github.com/npm/cli.git"
+    url = "https://registry.npmjs.org/npm/-/npm-9.30.0.tgz"
```

Comment:
I just had the version number wrong (it's actually 9.3.0) and 9.3.1 came out since I wrote this.  I updated that and switched everything to sha256, and kept the git url.